usaspending_api/common/api_request_utils.py

```diff
@@ -289,35 +290,71 @@ class AutoCompleteHandler():
     @staticmethod
     # Data set to be searched for the value, and which fields to look in
     # Mode is either "contains" or "startswith"
-    def get_values_and_counts(data_set, fields, value, mode="contains"):
+    def get_values_and_counts(data_set, filter_matched_ids, pk_name):
         value_dict = {}
         count_dict = {}
 
+        for field in filter_matched_ids.keys():
+            q_args = {}
```

Comment:
Right, I left it that way because I was working on another feature that I ended up taking out - this style was leftover from it.